main.js

```diff
@@ -294,6 +320,28 @@ export class IdDecoder {
         bitLength: this.fixedBitLength
       });
     }
+    if(this.multihash) {
+      // <varint hash fn code>: identity function
+      const [hashFnCode] = decoded;
+
+      if(hashFnCode !== MULTIHASH_IDENTITY_FUNCTION_CODE) {
+        throw new Error('Invalid multihash function code.');
+      }
+      // <varint digest size in bytes>: 32
+      const digestSize = decoded[1];
+
+      if(digestSize !== this.expectedSize) {
+        throw new Error('Unexpected identifier size.');
+      }
+
+      const bytes = decoded.subarray(2);
+      if(bytes.byteLength !== this.expectedSize) {
+        throw new Error(
+          `Invalid identifier size. Identifier must be ` +
```

Comment:
Should this say "decoded identifier" or similar?  Hopefully people don't hit this issue, but if they did, would it be confusing if it was the encoded or decoded length?

Reply:
Updated to say "decoded identifier" fb9cf6f